models/kyberswap/arbitrum/kyberswap_arbitrum_trades.sql

```diff
@@ -65,8 +65,31 @@ kyberswap_dex AS (
     {% else %}
     WHERE t.evt_block_time >= TIMESTAMP '{{ project_start_date }}'
     {% endif %}
-    
 
+    UNION ALL
+
+    SELECT
+        t.evt_block_time                                                                                          AS block_time
+        ,t.sender                                                                                                 AS taker
+        ,t.recipient                                                                                              AS maker
+        ,cast(if(starts_with(cast(t.deltaQty0 as varchar), '-'), abs(t.deltaQty1), abs(t.deltaQty0)) as uint256)  AS token_bought_amount_raw
+        ,cast(if(starts_with(cast(t.deltaQty0 as varchar), '-'), abs(t.deltaQty0), abs(t.deltaQty1)) as uint256)  AS token_sold_amount_raw
+        ,NULL                                                                                                     AS amount_usd
+        ,if(starts_with(cast(t.deltaQty0 as varchar), '-'), p.token1, p.token0)                                   AS token_bought_address
+        ,if(starts_with(cast(t.deltaQty0 as varchar), '-'), p.token0, p.token1)                                   AS token_sold_address
+        ,t.contract_address                                                                                       AS project_contract_address
+        ,t.evt_tx_hash                                                                                            AS tx_hash
+        ,'elastic_2'                                                                                              AS version
```

Comment:
Do you want to add some seed for trades with `elastic_2` version?

Reply:
No we can skip it, I just update the schema to pass the test.